config/initializers/datadog_apm.rb

```diff
@@ -12,9 +18,27 @@
     default_controller_service: 'samson-rails-controller',
     default_cache_service: 'samson-cache',
     default_database_service: 'samson-mysql',
+    distributed_tracing_enabled: true,
     trace_agent_hostname: ENV.fetch('STATSD_HOST', 'localhost'),
     tags: {
+      env: ENV['RAILS_ENV'],
       project: 'samson'
     }
   }
+
+  # Dalli instrumentation
+  Datadog::Monkey.patch_module(:dalli)
+  pin = Datadog::Pin.get_from(::Dalli)
+  pin.service = 'samson-memcached'
+
+  # Faraday instrumentation
+  Datadog::Monkey.patch_module(:faraday)
+  pin = Datadog::Pin.get_from(::Faraday)
+  pin.service = 'samson'
+
+  # SuckerPunch instrumentation
+  require 'sucker_punch'
```

Comment:
... I'll get rid of sucker-punch ... should never have been added :(

Reply:
... or maybe our job-execution should use sucker-punch ... idk ... but having 2 systems is weird :D